config/locales/gobierto_dashboards/views/ca.yml

```diff
@@ -44,11 +43,45 @@ ca:
           tenders_for: licitacions per un import de
         tender_amount: Imp. licitació
         title: Contractes i licitacions
+      subsidies:
+        amount: Import
+        amount_distribution: Distribució per imports
+        beneficiary: beneficiari
+        category: Categoria
+        date: Data
+        dates: Dates
+        description: Visualització i anàlisi de les subvencions concedides per %{entity_name}
+        fromto: De a Nº Subv.
+        main_beneficiaries: Principals beneficiaris
+        more: Més de
+        nav:
+          subsidies: Subvencions
+          summary: Resum
+        regulatory_bases: Bases reguladores
+        subsidies: Subvencions
+        summary:
+          collective_subsidies_for: a col·lectius per import de
+          individual_subsidies_for: a particulars per import de
+          label_half_spendings_subsidies: El <strong> <span id = "half-spendings-subsidies-pct">
+            </ span> </ strong> de subvencions concentren el 50% de tota la despesa
+          label_larger_subsidy_amount: La major subvenció suposa <strong> <span id
+            = "larger-subsidy-amount-pct"> </ span> </ strong> de tota la despesa
+            en subvencions
+          label_less_than_1000: El <strong> <span id = "less-than-1000-pct"> </ span>
```

Comment:
Back again?

Reply:
Still not sure about the best way to handle these, tbh. Separation is important, but it'll also be messier to handle.

Reply:
My point of view here is, let's imagine we move the translations to a professional tool, like transifex, why a translator have to know there are HTML in the text and not only this, but even the HTML contains required identificators for the app?

If the translation was: `label_larger_subsidy_amount: La <h1>major</h1> subvenció...`, it wouldn't be OK, but.... mmmm reasonable. But including identificadors? no way.

On the other hand, someone else is developing the controller and need to debug the values. Where are they? OMG, embedded in the translation texts. This is about [comparing apples and oranges](https://www.wordreference.com/es/en/translation.asp?spen=mezclar%20las%20churras%20con%20las%20merinas). Although split the sentence wouldn't be neither the best approach, at least the developer knows where its code is.

Reply:
I did the split, you may check now 👍